base/libgit2/types.jl

```diff
@@ -274,20 +274,20 @@ immutable DiffFile
     id::Oid
     path::Cstring
     size::Int64
-    flags::Cuint
```

Comment:
Yes, "incorrect" was too strong, I'll fix the commit message. This simply makes the types closer to C, so that the comparison is easier.